swift/swift.py

```diff
@@ -28,22 +28,31 @@ class Swift(QObject):
     Note that QApplication instance must be created before instantiating Swift object.
 
     Brief procedure:
-        1. Create buses.
-        2. Create apps.
-        3. Show frames of each app.
+        1. Load setup environment.
+        2. Create buses.
+        3. Create apps and show their frames.
+
+    Attributes:
+        _buses: A dictionary for storing buses.
+          Each element represents a bus.
+          A key is a bus name and its value is a Bus object.
+        _apps: A dictionary for storing apps.
+          Each element represents an app.
+          A key is an app name and its value is an App object.
+        _subscribers: A dictionary for storing subscribers.
+          Each element represents subscribers of each bus.
+          A key is a bus name and its value is a set containing its subscribers.
     """
 
-    def __init__(self, setup_env: dict, parent=None):
+    def __init__(self, setupEnv: dict, parent=None):
         """
         Args:
-            setup_env: A dictionary containing set-up environment about app and bus.
+            setupEnv: A dictionary containing set-up environment about app and bus.
               It has two keys; "app" and "bus".
-
-              In "app", there may be keys below; 
-                "path", "module", "class", "show", "pos", "bus", and "args".
-              In "bus", there may be keys below;
-                "timeout".
-
+              In "app", there are apps containing keys as below; 
+                "path" (optional), "module", "class", "show", "pos", "bus", and "args" (optional).
+              In "bus", there are buses containing keys as below;
+                "timeout" (optional).
               For details, see setup.json.

```

Comment:
I think this does not specify the dictionary structure enough.
For example, what are "apps" here:

> In "app", there are apps containing keys as below;

One may deduce that "apps" are `dict`s, but how are they stored in "app" section? Are they in a `list`?
Please modify the description so that a user can make a setup dictionary without the setup.json file.
Note that the setup.json file is NOT a part of the `swift` package.

Reply:
However, maybe this is not in the scope of this PR.
I will make a new issue from this comment, and resolve it in another PR (probably in the PR regarding to the `Info`s).

Reply:
Okay! I will leave it as it is.